src/js/components/Login.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import GoogleLogin from 'react-google-login';
+import FontAwesome from 'react-fontawesome';
+
+
+export default class Login extends React.Component {
+  constructor() {
+    super();
+    this.responseGoogle = this.responseGoogle.bind(this);
+  }
+
+  responseGoogle(response) {
```

Comment:
To avoid `bind(this)` in the constructor, define this as an arrow function like this...
```
responseGoogle = (response) => {
   // add your code here
}
```